aimsprop/plot.py

```diff
@@ -220,7 +220,7 @@ def plot_population(
     else:
         raise ValueError("Unknown time units: %s" % time_units)
 
-    if state_colors:
+    if len(state_colors) > 0:
```

Comment:
@hayleyweir can you confirm this will maintain the intended plotting behavior already in the package?

Reply:
Ah. I see, this is in response to the function signature changing. We probably want to keep this as it was previously :)